src/setup.rs

```diff
@@ -1,4 +1,27 @@
 use std::net::{Ipv4Addr, SocketAddrV4};
+use tokio::net::TcpListener;
+use tracing::{error, info, Level};
+use tracing_subscriber::FmtSubscriber;
+
+pub fn initialise_logging() {
+    let subscriber = FmtSubscriber::builder()
+        .with_max_level(Level::TRACE)
+        .finish();
+    tracing::subscriber::set_global_default(subscriber)
+        .expect("setting default tracing subscriber failed!");
+    info!("Response cannon spinning up...");
```

Comment:
```suggestion
    info!("Response cannon spinning up…");
```

Reply:
This is the command line, not a web interface. I'd avoid printing unicode.